src/Mod/OpenSCAD/OpenSCADCommands.py

```diff
@@ -188,6 +188,67 @@ def GetResources(self):
                 'MenuText': QtCore.QT_TRANSLATE_NOOP('OpenSCAD_MirrorMeshFeature', 'Mirror Mesh Feature...'),
                 'ToolTip' : QtCore.QT_TRANSLATE_NOOP('OpenSCAD_MirrorMeshFeature', 'Create Mirror Mesh Feature')}
 
+class ScaleMeshFeature:
+    def IsActive(self):
+        return FreeCADGui.Selection.countObjectsOfType('Mesh::Feature') > 0
+
+    def Activated(self):
+        import Part,OpenSCADFeatures,OpenSCADUtils
+        selection=FreeCADGui.Selection.getSelectionEx()
+        for selobj in selection:
+            newobj=selobj.Document.addObject("Mesh::Feature",'scale')
+            newobj.Label='scale_%s' % selobj.Object.Label
+            msh=selobj.Object.Mesh
+            items=["[1;1;1]"]
+            item, ok = QtGui.QInputDialog.getItem(QtGui.QApplication.activeWindow(),u'Scale about which Axis?',u'Enter scaling value:',items,editable=True)
+            if ok:
+                splits = list(item.replace('[','').replace(']','').split(';'))
+                x = float(splits[0])
+                y = float(splits[1])
+                z = float(splits[2])
+                vec = FreeCAD.Base.Vector(x,y,z)
+                newmesh=OpenSCADUtils.scalemesh(msh, vec)
+                newobj.Mesh=newmesh
+                selobj.Object.ViewObject.hide()
+            else:
+                selobj.Document.removeObject(newobj.Name)
+        FreeCAD.ActiveDocument.recompute()
+    def GetResources(self):
+        return {'Pixmap'  : 'OpenSCAD_ScaleMeshFeature',
+                'MenuText': QtCore.QT_TRANSLATE_NOOP('OpenSCAD_ScaleMeshFeature', 'Scale Mesh Feature...'),
+                'ToolTip' : QtCore.QT_TRANSLATE_NOOP('OpenSCAD_ScaleMeshFeature', 'Create Scale Mesh Feature')}
+
+
+class ResizeMeshFeature:
+    def IsActive(self):
+        return FreeCADGui.Selection.countObjectsOfType('Mesh::Feature') > 0
+
+    def Activated(self):
+        import Part,OpenSCADFeatures,OpenSCADUtils
```

Comment:
Neither `Part` nor `OpenSCADFeatures` seem to be used, so why import them? Surely `OpenSCADUtils` was already imported at the beginning of the file, no?